pkg/scheduler/api/v1/types.go

```diff
@@ -169,11 +195,11 @@ type ExtenderConfig struct {
 	// Verb for the bind call, empty if not supported. This verb is appended to the URLPrefix when issuing the bind call to extender.
 	// If this method is implemented by the extender, it is the extender's responsibility to bind the pod to apiserver. Only one extender
 	// can implement this function.
-	BindVerb string
+	BindVerb string `json:"bindVerb,omitempty"`
```

Comment:
for posterity, this remains compatible because a custom unmarshaler was implemented to remain case insensitive